packages/twenty-front/src/modules/ui/layout/dropdown/components/Dropdown.tsx

```diff
@@ -186,6 +187,7 @@ export const Dropdown = ({
             onDropdownClose={onClose}
             onDropdownOpen={onOpen}
           />
+          <DropdownCloseOnCommandMenuShortcutEffect />
```

Comment:
I'm not convinced by this abstraction, Dropdown shouldn't be aware of its implementations.

We should find a way to have the command menu closing any of its open dropdown by a synchronous hook call, because this is the way we want to go, instead of relying on effects in the dropdown, which we already tried and removed, let's not put more effect in the dropdown component and find a synchronous solution.